### ISSUE TITLE
[Feature]: Comments in the URL list text file

### ISSUE THREAD

**Issue by s100** [IBMa/equal-access#2321]
### Project

accessibility-checker for Node

### Description

Right now you can provide a list of URLs to check, in a text file. But for us, some of those URLs are (temporarily) failing the accessibility check due to an upstream bug. I'd like to be able to temporarily disable these URLs, *without* just completely removing them from the text file. I'd also like to be able to add a comment linking to the upstream bug and explaining the circumstances under which these URLs can be re-enabled.
